charts/kubearchive/values.yaml

```diff
@@ -53,12 +58,14 @@ operator:
 database:
   # Controls if the database is deployed or not. The secret with the credentials
   # is deployed always, as the sink and the api-server need to connect somewhere.
-  # Provide the proper `url` and credentials if you provide your database
-  # TODO: set 'enabled' to 'false' when we start distributing the chart for production installation
+  # If 'enabled: true' the secret credentials are automatically set, if you want to
+  # provide your own database, set 'enabled: false' and modify the secret after deploying
   enabled: true
 
   name: "{{ .Release.Name }}-database"
-  url: "{{ tpl .Values.database.name . }}.{{ .Release.Namespace }}.svc.cluster.local"
+
+  url: "kubearchive-database.kubearchive.svc.cluster.local"
```

Comment:
Why are we including this change?

According to the way that [Kubernetes set up the DNS for the different services](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#services), this name is built through the following template:

<service-name>.<service-namespace>.svc.cluster.local

If we change the namespace provided in the `helm template`/`helm install` command (e.g. if we want to deploy 2 different kubearchive charts for developing/debugging/troubleshooting purposes), we need to remember also updating this line.

Reply:
Ops, I did oversee this change when reviewing my own changes. I reverted it so there is no change here.